include/hobbes/util/func.H

```diff
@@ -15,6 +15,13 @@ template <typename R, typename ... Args>
     typedef R result_type;
   };
 
+template <typename R, typename ... Args>
```

Comment:
nice! i did same thing in my app code :+1:

Reply:
I guess I didn't hit it until recently.  I try to fix up these little things as I hit them.  :T